libsolidity/ast/Types.cpp

```diff
@@ -3694,7 +3694,7 @@ MemberList::MemberMap TypeType::nativeMembers(ASTNode const* _currentScope) cons
 			auto bases = contract.annotation().linearizedBaseContracts;
 			solAssert(bases.size() >= 1, "linearizedBaseContracts should at least contain the most derived contract.");
 			// `sliced(1, ...)` ignores the most derived contract, which should not be searchable from `super`.
-			for (ContractDefinition const* base: bases | boost::adaptors::sliced(1, bases.size()))
+			for (ContractDefinition const* base: bases | ranges::views::drop(1))
```

Comment:
```suggestion
			for (ContractDefinition const* base: bases | ranges::views::tail)
```

Reply:
OK I'll push it in a minute.
to me both are the same,  why do you prefer `tail`?
I admit it looks nicer though.

Reply:
I just like tail more because it's more obvious that `drop(1)`. With `drop`, I'll have to look up if it drops from the beginning or the end.